cwms-http-client/src/main/java/mil/army/usace/hec/cwms/http/client/ApiConnectionInfoFactory.java

```diff
@@ -0,0 +1,18 @@
+package mil.army.usace.hec.cwms.http.client;
+
+public class ApiConnectionInfoFactory {
```

Comment:
This should be final unless your class design intentionally includes support for sub-classes

Reply:
Devin is changing this to final